src/components/Country.jsx

```diff
@@ -1,10 +1,21 @@
 import React from 'react'
 
-const Country = () => {
+const Country = ({ name, flag, population, region, capital, onClick }) => {
   return (
```

Comment:
Good work on destruct the props,  by doing this you are flattening your data and sending the most simple props to your componenet